src/main/java/org/osc/manager/rest/server/api/ManagerConnectorApis.java

```diff
@@ -68,20 +68,20 @@ public void createManagerConnectorDb(EntityManager em, TransactionControl txCont
      * @throws Exception
      */
     @POST
-    public ApplianceManagerConnectorEntity createManagerConnector(ApplianceManagerConnectorEntity entity)
+    public String createManagerConnector(ApplianceManagerConnectorEntity entity)
```

Comment:
I see you update this one to return the id which is good. The same API patterns should be used everywhere. Consistent API behavior is highly important: creates return the id and bad request if you create something that already exist (i.e.: unique names), updates return the updated entity and 404 not found if the id is not found, getlist returns all the ids and empty list if nothing is found, 404 if a parent element to the list is not found (i.e.: domain to policies, 404 if domain id is not found), getbyid returns null if the entity is not found, deletes dont return anything and succeed silently if the targeted id is not found.

Reply:
ok